install-guides/README.md

```diff
@@ -28,7 +28,9 @@ In kubeadm these variables can be set in a file that sits in
 ```
 KUBELET_EXTRA_ARGS=\
 --cpu-manager-policy=static \
----reserved-cpus=0 \
+--reserved-cpus=0 \
+--memory-manager-policy=Static \
+--reserved-memory 0:memory=100Mi \
```

Comment:
I expect the config to be more complex and more machine-dependent if compared to what is required for CPUs.
I've been using a template like
```
    memoryManagerPolicy: "Static"
    evictionHard:
      memory.available: "${EVICTION_THRES}Mi"
    kubeReserved: 
      memory: "${KUBE_RES}Mi"
    reservedMemory:
      - numaNode: ${NUMA_NODE}
        limits:
          memory: "(${KUBE_RES} + ${SYSTEM_RES} + ${EVICTION_THRES})Mi"
    systemReserved: 
      memory: "${SYSTEM_RES}Mi"
```
Maybe @cynepco3hahue can suggest a minimal valid configuration, and we can link back to the kube docs for more detailed, always up-to-date recommendations?